packages/server/graphql/public/permissions.ts

```diff
@@ -61,7 +63,7 @@ const permissionMap: PermissionMap<Resolvers> = {
     SAMLIdP: rateLimit({perMinute: 120, perHour: 3600})
   },
   Organization: {
-    saml: and(isViewerBillingLeaderSource, isOrgTierSource('enterprise'))
+    saml: and(isViewerBillingLeader('source.id'), isOrgTier('source.id', 'enterprise'))
```

Comment:
curious on your thoughts about this new pattern. 
I want to reuse rules as much as I can. For example, this rule requires an orgId to determine if the viewer is on the org. That `orgId` may exist in an arg, or in the source, and it could have any name. I think a dot notation path is the cleanest way to do this, but wonder what you think?

Reply:
I like it! Not having type safety is my main concern, but that's still probably safer that accidentally mixing `isOrgTierSource` for `isOrgTier`